spec/signup_error_handler_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'signup_error_handler'
+
+describe SignupErrorHandler do
+  context "when passed an error during sign-up validation" do
+    let(:errors) {
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)